lib/platform-stats.js

```diff
@@ -199,3 +344,121 @@ export const updateTopMeasurementParticipants = async (createPgClient) => {
     Sentry.captureException(err)
   }
 }
+
+/**
+ * @param {import('pg').Client} pgClient
+ */
+export const aggregateAndCleanupRecentData = async (pgClient) => {
```

Comment:
```suggestion
export const aggregateAndCleanUpRecentData = async (pgClient) => {
```

"to cleanup" doesn't exist :)

Reply:
Also needs to be renamed above